docs(tutorials): content validation - MTA-1594 (#229)
scaleway/docs-content: tutorials/deploy-nextcloud-s3/index.mdx

- You have an account and are logged into the [Scaleway Console](https://console.scaleway.com)
- You have [configured your SSH Key](/console/my-project/how-to/create-ssh-key/)

> - You have a compute instance running on Ubuntu Focal Fossa (20.04)

Reviewer: ```suggestion
- You have an Instance running on Ubuntu Focal Fossa (20.04)
```

- You have [installed and configured](/tutorials/mariadb-ubuntu-bionic/) MariaDB on your Virtual Instance
- You have sudo privileges or access to the root user.

</Message>

## Installing Nextcloud

> 1. Login to your server via SSH.

Reviewer 2: ```suggestion
1. Log in to your server via SSH.
```
